Expensify/App
# Simplify the ready state of the welcome task

### Details
This was a simplification I noticed in the code

### Fixed Issues
None

### Tests & QA
1. Log into chat as a brand new user that is not a member of any policies or workspaces
2. Verify the create menu is open
